jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -107,44 +114,22 @@ public String toString() {
     return contextAsString();
   }
 
-  public static JaegerSpanContext contextFromString(String value)
```

Comment:
may want to move this particular change into a separate PR.

Reply:
Sure.